tests/containers/helm.pm

```diff
@@ -93,7 +93,11 @@ sub run {
 
     assert_script_run("helm install helm-test-$job_id ~/helm-test/ --values ~/helm-test/values.yaml --set job_id=$job_id");
     assert_script_run("helm list");
-    my $pod = script_output('kubectl get pods -o name --no-headers=true | grep helm-test-$job_id');
+    
+    my $getpod = 'kubectl get pods -o name --no-headers=true | grep helm-test-$job_id';
+    script_retry($getpod, retry => 3, delay => 10);
```

Comment:
you can directly use `script_output_retry`
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L1746

Reply:
thanks, done, testing